doc/sphinx/addendum/rewrite-rules.rst

```diff
@@ -0,0 +1,144 @@
+.. _rewrite_rules:
+
+User-defined rewrite rules
+==========================
+
+.. warning::
+
+   The status of rewrite rules is highly experimental.
```

Comment:
```suggestion
   Rewrite rules are highly experimental.
```

Reply:
I went for the wording used in the doc about SProp

Reply:
SProp doesn't follow the doc convention, so not a good model to follow

Reply:
You convinced me, I changed to your suggestion